lib/nerves_hub_link/socket.ex

```diff
@@ -299,7 +299,7 @@ defmodule NervesHubLink.Socket do
 
   defp stop_iex(%{assigns: %{iex_pid: iex}} = socket) do
     _ = Process.unlink(iex)
-    :ok = GenServer.stop(iex, 10_000)
+    :ok = GenServer.stop(iex, :normal)
```

Comment:
Great find!

```suggestion
    GenServer.stop(iex, :normal, 10_000)
```
I suspect that the intention was to override the timeout on stop so it wouldn't be `:infinity`. I also don't think the `:ok` is needed since I don't think `GenServer.stop/3` can return anything else.